DEVELOPMENT.md

```diff
@@ -76,21 +76,21 @@ make test-unit
 
 # Run the docs
 
-To get started with developing the docs we provide a docker image which you can use from within the `/docs` directory. It should work on all platforms. To get it up and running:
+To get started with developing the docs we provide a docker image which you can use with the `/docs` directory. It should work on all platforms. To get it up and running:
 
 ```
+make docs
 docker run -it --rm \
         --name hugo-server \
         -p 1313:1313 \
-        -v $(PWD):/src:cached \
+        -v ${PWD}/docs:/src:cached \
```

Comment:
I assume this was originally written on a case insensitive operating system. On linux there's no command `PWD`, so executing a command with `$(PWD)` returns "command not found: PWD".

So there are two options, either use the shell variable `${PWD}`, or call the `$(pwd)` command. They both result in the same output.

---
As for adding `/docs` into the command. This means the docker command should be run from the project root, rather than from within the docs folder. It remove one (small) step. Simplifying getting started.

Reply:
Also calling `make docs` means that you have the `gomods/hugo` container available.

Reply:
Yep! Can you make this clear as a note in the docs ?
`As for adding /docs into the command. This means the docker command should be run from the project root, rather than from within the docs folder. It remove one (small) step. Simplifying getting started.`

The reason being it might be hard to understand what PWD would mean, for a few it may mean the root directory, but for others it may mean `docs/`. It is just a small change, but I think being explicit is clear as that is the only difference between the two changes.

Reply:
Oohh! haha yes! Good thinking. I'll add it :)